src/components/structures/RoomView.js

```diff
@@ -748,6 +754,21 @@ export default createReactClass({
             }
         }
     },
+    onEventDecrypted(ev) {
+    if (!SettingsStore.getValue('dontShowChatEffects')) {
+        if (ev.isBeingDecrypted() || ev.isDecryptionFailure() ||
+            this.state.room.getUnreadNotificationCount() === 0) return;
+        this.handleConfetti(ev);
+    }
+    },
+    handleConfetti(ev) {
+        if (this.state.matrixClientIsReady) {
+            const messageBody = _t('sends confetti');
+            if (isConfettiEmoji(ev.getContent()) || ev.getContent().body === messageBody) {
```

Comment:
So one issue with this is it will not trigger if you have a different UI language from the other user, I suggest just sticking a 🎉 at the end of `sends confetti` for simplicity and use a custom msgtype for `/confetti custom message text`

Reply:
yes this was added before our conversation yesterday. i will try that before!. thanks for the idea though!